cdap-ui/app/features/dashboard/templates/partials/addwdgt.html

```diff
@@ -23,7 +23,7 @@ <h4 class="modal-title pull-left">Add a Widget</h4>
 
           <div class="form-group">
             <label class="col-xs-3 control-label">Metric</label>
-            <div class="col-xs-9">
+            <div class="col-xs-8 modal-body">
```

Comment:
This is not 'semantically' correct. ```modal-body``` class is something given by bootstrap and we should use it to enclose everything that we put under modal body.